packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -240,6 +240,48 @@ const getLocation = (
   return ''
 }
 
+/*const getSavedApplications = (props: IFullProps): IApplicationData => {
```

Comment:
@soumita-dsi is this meant to be commented out?

Reply:
Sorry, I forgot to remove this commented part. I will remove that.